packages/server/dataloader/customLoaderMakers.ts

```diff
@@ -955,3 +964,59 @@ export const publicTemplatesByType = (parent: RootDataLoader) => {
     }
   )
 }
+
+export const allFeatureFlags = (parent: RootDataLoader) => {
```

Comment:
+1 if this will only be used by dataloaders you could make the PG query a little more efficient by doing it all in 1 go in the `allFeatureFlagsByOwner`, but probably not a big deal until we benchmark